.github/gha_common.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+
+# Exit on error.
+set -o errexit
+# Exit on error inside any functions or subshells.
+set -o errtrace
+# Do not allow use of undefined vars. Use ${VAR:-} to use an undefined VAR
+set -o nounset
+
+GREEN_COLOR="\033[1;32m"
+YELLOW_COLOR="\033[1;33m"
+NO_COLOR="\033[0m"
+
+terraform_init(){
+    if [[ $GITHUB_ACTIONS ]]
+    then
+        terraform init -backend=false
+    fi
+}
```

Comment:
Can we have a [newline](https://gist.github.com/camh-/1bebfcff1b0f814e9b191edc60d5206b) at the end of the files that are giving this warning

Reply:
Added a line now @umair-fayaz